apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -24,6 +23,232 @@ namespace Google.Cloud.Bigtable.V2
     public partial class BigtableClient
     {
         /// <summary>
+        /// Mutates a row atomically based on the output of a predicate Reader filter.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// Note that string is implicitly convertible to <see cref="BigtableByteString"/>, so <paramref name="rowKey"/> can
+        /// be specified using a string as well and its UTF-8 representations will be used.
+        /// </para>
+        /// <para>
+        /// This method simply delegates to <see cref="CheckAndMutateRow(CheckAndMutateRowRequest, CallSettings)"/>.
+        /// </para>
+        /// </remarks>
+        /// <param name="tableName">
+        /// The unique name of the table to which the conditional mutation should be
+        /// applied. Must not be null.
+        /// </param>
+        /// <param name="rowKey">
+        /// The key of the row to which the conditional mutation should be applied.
+        /// Must not be empty.
+        /// </param>
+        /// <param name="predicateFilter">
+        /// The filter to be applied to the contents of the specified row. Depending
+        /// on whether or not any results are yielded, either <paramref name="trueMutations"/> or
+        /// <paramref name="falseMutations"/> will be executed. If null, checks that the row contains
+        /// any values at all.
+        /// </param>
+        /// <param name="trueMutations">
```

Comment:
Document that true/false can't *both* be null/empty?

Reply:
This line for the param points this out: `Must contain at least one entry if <paramref name="falseMutations"/> is null or empty"`. Similar for `falseMutations`. Do you think something else should be added to the remarks?